tests/queries/0_stateless/02900_add_subtract_interval_with_string_date.sql

```diff
@@ -109,3 +109,31 @@ SELECT subtractSeconds(materialize('2023-10-22'), materialize(1)), subtractSecon
 SELECT subtractMilliseconds(materialize('2023-10-22'), materialize(1)), subtractMilliseconds(materialize('2023-10-22 00:00:00.000'), materialize(1));
 SELECT subtractMicroseconds(materialize('2023-10-22'), materialize(1)), subtractMicroseconds(materialize('2023-10-22 00:00:00.000'), materialize(1));
 SELECT subtractNanoseconds(materialize('2023-10-22'), materialize(1)), subtractNanoseconds(materialize('2023-10-22 00:00:00.000'), 1);
+
+SELECT '-- plus operator';
+
+SELECT '2023-10-23' + INTERVAL 1 YEAR;
```

Comment:
Do we also support parsing of Datetime64 with subsecond precision?

Reply:
We do, I'll add tests.